plugins/fetcher.cc

```diff
@@ -113,9 +112,16 @@ struct FetcherFunction: public ValueFunctionT<FetcherArgs, FetcherOutput> {
                 blob = CellValue::blob(streamo.str());
             }
 
-            result.content = ExpressionValue(std::move(blob), info.lastModified);
+            FsObjectInfo info = stream.info();
```

Comment:
It still doesn't work. The exception is caught somewhere within boost::iostreams and never makes it out.